ReactAndroid/src/main/java/com/facebook/react/packagerconnection/PackagerConnectionSettings.java

```diff
@@ -54,10 +54,21 @@ public String getDebugServerHost() {
   }
 
   public void setDebugServerHost(String host) {
+    mDebugServerHost = host;
     mPreferences.edit().putString(PREFS_DEBUG_SERVER_HOST_KEY, host).apply();
   }
 
+  public void setDebugServerHostWithoutPersisting(final String host, final boolean savePrefs) {
```

Comment:
"savePrefs" is not used anywhere, should delete it?

Reply:
@sshic good catch. sorry for missing this and updated 😅